contracts/src/v0.8/vendor/Ownable.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @dev Contract module which provides a basic access control mechanism, where
+ * there is an account (an owner) that can be granted exclusive access to
+ * specific functions.
+ *
+ * This module is used through inheritance. It will make available the modifier
+ * `onlyOwner`, which can be aplied to your functions to restrict their use to
+ * the owner.
+ *
+ * This contract has been modified to remove the revokeOwnership function
+ */
+contract Ownable {
```

Comment:
Can we use our ConfirmedOwner instead?

Reply:
sure - curious what the benefit of the ConfirmedOwner is?

Reply:
spoke offline - just safer